1.6/Dockerfile.rhel7

```diff
@@ -36,18 +36,18 @@ EXPOSE 8080 50000
 #RUN  curl -L -o /tmp/ssh-credentials.hpi http://updates.jenkins-ci.org/latest/ssh-credentials.hpi
 #RUN  curl -L -o /tmp/credentials.hpi http://updates.jenkins-ci.org/latest/credentials.hpi
 
-RUN yum install -y tar zip unzip java-1.7.0-openjdk jenkins-1.609.1 && \
-    yum clean all  && \
-    chmod og+rw /var/lib/jenkins && \
-    curl -L https://github.com/openshift/origin/releases/download/v1.0.2/openshift-origin-v1.0.2-325c7b7-linux-amd64.tar.gz | tar -zx && \
-    mv oc /usr/local/bin && \
-    rm oadm && \
-    rm openshift
-
 COPY run.sh /usr/local/bin/run.sh
 COPY ./contrib/ /opt/openshift
 
-RUN chmod -R og+rw /opt/openshift
+RUN yum install -y tar zip unzip java-1.7.0-openjdk jenkins-1.609 && \
+  yum clean all  && \
+  mkdir /var/lib/jenkins && \
+  chmod og+rw /var/lib/jenkins && \
+  chmod -R og+rw /opt/openshift && \
+  curl -L https://github.com/openshift/origin/releases/download/v1.0.2/openshift-origin-v1.0.2-325c7b7-linux-amd64.tar.gz | tar -zx && \
+  mv oc /usr/local/bin && \
+  rm oadm && \
+  rm openshift
```

Comment:
Sorry for nit-picking, but can't the indentation stay 4 spaces? It IMHO looks nicer (since "RUN " has 4 chars) and would be consistent with our other images (and the CentOS Dockerfile). Plus it'd be easier to review.